src/shared/components/form/formSelect/formSelect.js

```diff
@@ -0,0 +1,39 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+
+export default class FormSelect extends Component {
+  buildOptions = () => {
+    const { prompt, options } = this.props;
+    const opts = [];
+
+    if (prompt) {
+      opts.push(<option key="prompt" value="">{ prompt }</option>);
+    }
+    options.forEach(option => opts.push(<option key={option.value} value={option.value}>{option.label}</option>));
+    return opts;
+  }
+
+  render() {
+    return (
+      <select id={this.props.id} onChange={this.props.onChange}>
+        {this.buildOptions()}
+      </select>
+    );
+  }
```

Comment:
When I select an option from the FormSelect component, I'm not seeing the selected option displayed inside of the select.  Seeing this on the Signup page, and also the Mentor Request page so far.  Do we need to add some code to the `<select>` element here that sets a value (possibly with a passed in prop, as a result of the onChange handler updating the state in the parent component) ?